src/platforms/unity/configuration/offline-caching.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Offline Caching
```

Comment:
should this be its own page? usually we only make it available in the basic options
https://docs.sentry.io/platforms/android/configuration/options/#cache-directory-path

and also in the shutdown & draining page https://docs.sentry.io/platforms/android/configuration/draining/

Reply:
Thanks for pointing that out. I completely missed it.